src/components/flow/actions/sendemail/SendEmailForm.tsx

```diff
@@ -356,9 +406,9 @@ export default class SendEmailForm extends React.Component<ActionFormProps, Send
     let message = '';
     let isValid = true;
     const file = files[0];
-    const fileExtension = file.name.split('.').pop();
+    const fileExtension = file.type.split('/')[0];
 
-    if (UNSUPPORTED_EMAIL_ATTACHMENTS.includes(fileExtension)) {
+    if (!['audio', 'video', 'image'].includes(fileExtension)) {
```

Comment:
For sending an email with attachments, I think we should accept all files except this list of extensions: https://github.com/greatnonprofits-nfp/floweditor/pull/38/files#diff-f88ed0b807262cd475ff365a82ce8120L30, because .pdf, .docx, .xlsx should be acceptable as well at this flow step.

Reply:
I've reverted this line to the previous version.